frontend/src/Components/WebSocket/WebSocket.tsx

```diff
@@ -22,6 +22,8 @@ const initialWebSocketState: WebSocketState = {
 };
 
 const initialLoginData: WebSocketLoginData = { user: '', session: '' };
+const baseRetryWait = 200;
```

Comment:
Do we want to use all caps consistently for such constants?
```suggestion
const BASE_RETRY_WAIT = 200;
```